api/app/settings/local.py

```diff
@@ -9,3 +9,9 @@
 MIDDLEWARE.extend(["debug_toolbar.middleware.DebugToolbarMiddleware"])
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+
+SWAGGER_SETTINGS["USE_SESSION_AUTH"] = False
+
+# Allow admin login with username and password
+ENABLE_ADMIN_ACCESS_USER_PASS = False
```

Comment:
Shouldn't this be `True` ?

Reply:
Defaulting to False seems more sensible from a security point of view.

Reply:
This is the local settings file - it's kinda the whole point of this PR.